tests/end2end-tests/data_process/gen_multi_task_remap_test.py

```diff
@@ -105,7 +105,7 @@ def main(args):
                 },
                 {
                     # will be ignored
-                    "edge_classification": {
+                    "reconstruct_edge_name": {
```

Comment:
I see the pipeline is still down. May you double check what is wrong here?

Reply:
Fixed